### PR TITLE
docker deploy stack command

### PR DESCRIPTION
**Pull request by myfreax**
Signed-off-by: huangyanxiong <huangyanxiong2013@gmail.com>

**- What I did**
specify a name for the docker deploy stack command
**- How I did it**
Replace `docker deploy vossibility-stack` with `docker deploy --bundle-file vossibilitystack.dab vossibility` in experimental/docker-stacks-and-bundles.md
**- How to verify it**
:eyes:
**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: experimental/docker-stacks-and-bundles.md [hunk 1]
```diff
@@ -52,15 +52,15 @@ Options:
...
+# docker deploy --bundle-file vossibilitystack.dab vossibility
 Loading bundle from vossibility-stack.dab
 Creating service vossibility-stack_elasticsearch
 Creating service vossibility-stack_kibana
 Creating service vossibility-stack_logstash
 Creating service vossibility-stack_lookupd
 Creating service vossibility-stack_nsqd
 Creating service vossibility-stack_vossibility-collector
-```
+```s
```
Comment (AkihiroSuda) [inline experimental/docker-stacks-and-bundles.md]: typo?

**Review by AkihiroSuda — CHANGES_REQUESTED** [review]

**Comment by AkihiroSuda** [comment]
Also, please consider carrying https://github.com/docker/docker/pull/30016 with https://github.com/docker/docker/pull/30016#pullrequestreview-15862225 ?

(cc @lixiaobing10051267 )

File: experimental/docker-stacks-and-bundles.md (hunk 1 above)
Comment (huangyanxiong01) [inline experimental/docker-stacks-and-bundles.md]: sorry,I have already updated

File: experimental/docker-stacks-and-bundles.md [hunk 2]
```diff
@@ -52,7 +52,7 @@ Options:
 Let's deploy the stack created before:
 
 ```bash
-# docker deploy vossibility-stack
+# docker deploy --bundle-file vossibilitystack.dab vossibility
```
Comment (thaJeztah) [inline experimental/docker-stacks-and-bundles.md]: The `--bundle-file` doesn't match the output below, which says:

```
Loading bundle from vossibility-stack.dab
```

Also, the _name_ given (see the output below) is `vossibility-stack`, not `vossibility`


So this should be:

```bash
$ docker deploy --bundle-file vossibility-stack.dab vossibility-stack
```

**Review by thaJeztah — CHANGES_REQUESTED** [review]
Let a comment, also, please squash your commits

**Comment by vdemeester** [comment]
Carrying into #30789

File: experimental/docker-stacks-and-bundles.md (hunk 2 above)
Comment (huangyanxiong01) [inline experimental/docker-stacks-and-bundles.md]: Thank you for correcting

**Pull request closed without merging**